ms2deepscore/data_generators.py

```diff
@@ -179,10 +176,10 @@ def _set_generator_parameters(self, **settings):
                 settings[key] = defaults[key]
         assert 0.0 <= settings["augment_removal_max"] <= 1.0, "Expected value within [0,1]"
         assert 0.0 <= settings["augment_removal_intensity"] <= 1.0, "Expected value within [0,1]"
-        if settings["use_fixed_set"] and settings["num_turns"] != 1:
```

Comment:
Now that I think of it, this should indeed work (I was worried that the 'caching' mechanism would fail)